src/vivarium_cluster_tools/psimulate/worker/vivarium_work_horse.py

```diff
@@ -169,7 +182,7 @@ def parameter_update_format(job_parameters: JobParameters) -> dict:
         },
         "randomness": {
             "random_seed": job_parameters.random_seed,
-            "additional_seed": job_parameters.input_draw,
+            "additional_seed": job_parameters.input_draw,  # <- FIXME: is this a typo?
```

Comment:
@rmudambi I didn't change this but it seemed fishy that we're setting additional seed to input_draw here.

I also think this is the thing that still has us having multiple columns of the same seed and draw in the new metadata file:
```
>>> [c for c in metadata2.columns if "draw" in c or "seed" in c]
['input_data.input_draw_number', 'run_configuration.run_key.input_draw', 'run_configuration.run_key.random_seed', 'randomness.random_seed', 'randomness.additional_seed']
```